packages/bootstrap-chat/src/FormResponse.jsx

```diff
@@ -129,6 +129,14 @@ class FormResponse extends React.Component {
               }}
             />
           </Col>
+          onKeyPress={(e) => {
+            console.log(e);
+            if (e.key === "Enter") {
+              tryMessageSend();
+              e.stopPropagation();
+              e.nativeEvent.stopImmediatePropagation();
+            }
+          }}
```

Comment:
This prop should be on the FormControl component, right now it's dangling not attached to any component